test/cts/README.md

```diff
@@ -0,0 +1,23 @@
+# Contribute CTS test case
+The tests in `./test/cts/test` folder were converted from original Android CTS test, for new created CTS test file, please add them into `./test/cts/test_supplement` folder
```

Comment:
@cuiyanx Please move this README file into its parent  directory 'test'. And modify **./test/cts/test** to **./cts/test**, and also modify **./test/cts/test_supplement** to **./cts/test_supplement**